libs/blocks/featured-article/featured-article.js

```diff
@@ -1,6 +1,16 @@
 import { getMetadata, createTag, getConfig } from '../../utils/utils.js';
-// import fetchTaxonomy from '../../scripts/taxonomy.js';
-// import { updateLinkWithLangRoot } from '../../utils/helpers.js';
+import fetchTaxonomy from '../../scripts/taxonomy.js';
+import { updateLinkWithLangRoot } from '../../utils/helpers.js';
+
+function createCategoryLink(doc, categoryEl) {
+  const category = getMetadata('article:tag', doc);
+  return fetchTaxonomy(getConfig(), '/topics')
+    .then((taxonomy) => {
+      const categoryTaxonomy = taxonomy.get(category) || taxonomy.get('News');
```

Comment:
I see that previously the fallback was just `'News'`, skipping the `taxonomy.get` call, is it actually necessary?

Reply:
Yes, this was addressed via Chris P comments in the resolved comment.

>if categoryTaxonomy === 'News' then categoryTaxonomy.link && categoryTaxonomy.name will be undefined